### ISSUE TITLE
Something broke in 0.3.1

### ISSUE THREAD

**Issue by ayourk** [uPesy/easyeda2kicad.py#31]
Love that you fixed issues so quickly.  Not sure what happened here:
```
easyeda2kicad --full --overwrite --lcsc_id=C2891863   
Traceback (most recent call last):
  File "/home/code/.local/bin/easyeda2kicad", line 8, in <module>
    sys.exit(main())
TypeError: main() missing 1 required positional argument: 'argv'
```

**Comment by ayourk** [comment]
I think I found the problem:
https://github.com/uPesy/easyeda2kicad.py/commit/bea9b18cd1821fd2ebce773482547d94c2cc8020#r74005847

**Comment by uPesy** [comment]
Fix in PR #32
